### PR TITLE
the code was different from the demo blow

### PR DESCRIPTION
**Pull request by qibing147147**
### Description
The code was different from the demo blow.

So I fix the issue, and copy the new code from demo.

And  create a pull request in mdn/html-examples,I hope you can review the code.

### Related issues and pull requests
[Example custom select is not available on the keychain #22513](https://github.com/mdn/content/issues/22513)

### CODE REVIEW

File: files/en-us/web/html/element/select/index.md [hunk 1]
```diff
@@ -204,14 +204,14 @@ This example basically:
...
-    <select name="1A" id="select" autocomplete="off" required>
+    <select name=1A id=select autocomplete=off required>
       <option>Carrots</option>
       <option>Peas</option>
       <option>Beans</option>
       <option>Pneumonoultramicroscopicsilicovolcanoconiosis</option>
     </select>
   </fieldset>
-  <fieldset id="custom">
+  <fieldset id=custom>
```
Comment (estelle) [inline files/en-us/web/html/element/select/index.md:214]: leave the quotes in

File: files/en-us/web/html/element/select/index.md [hunk 2]
```diff
@@ -227,7 +227,7 @@ This example basically:
 
 ```css
 body {
-  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif
```
Comment (estelle) [inline files/en-us/web/html/element/select/index.md:230]: on MDN we always include semi colons in our CSS

File: files/en-us/web/html/element/select/index.md [hunk 3]
```diff
@@ -239,36 +239,35 @@ html body form fieldset#custom div.select[data-multiple] div.header {
...
   align-content: center;
   justify-content: center;
   justify-items: center;
   align-items: center;
-  padding: 0.5em;
+  padding: .5em;
 }
 
-html body form fieldset#custom div.select div.header:hover::after {
+html body form fieldset#custom div.select div.header:hover:after {
```
Comment (estelle) [inline files/en-us/web/html/element/select/index.md:264]: pseudo elements use double colon notation

File: files/en-us/web/html/element/select/index.md [hunk 4]
```diff
@@ -314,6 +316,8 @@ html body form fieldset#custom div.select[data-open] datalist {
 }
 
 html body form fieldset#custom div.select datalist {
+  -webkit-appearance: none;
+  -moz-appearance: none;
```
Comment (estelle) [inline files/en-us/web/html/element/select/index.md:320]: no needed

File: files/en-us/web/html/element/select/index.md [hunk 5]
```diff
@@ -336,30 +340,26 @@ html body form fieldset#custom div.select datalist div.option {
...
 }
 
-html body form fieldset#custom div.select datalist div.option:hover,
-html body form fieldset#custom div.select datalist div.option:focus,
+html body form fieldset#custom div.select datalist div.option:hover {
+  background-color: blue;
+  color: white;
+}
+
+html body form fieldset#custom div.select datalist div.option:focus {
```
Comment (estelle) [inline files/en-us/web/html/element/select/index.md:348]: not sure why this is separated out.

File: files/en-us/web/html/element/select/index.md [hunk 6]
```diff
@@ -482,56 +482,49 @@ for (const select of selects) {
...
     return div.offsetWidth;
   }));
-
-  console.log(width);
-  div.style.width = `${width}px`;
+  console.log(width)
+  div.style.width = width + 'px';
 }
-document.forms[0].onsubmit = (e) => {
+document.forms[0].onsubmit = function (e) {
```
Comment (estelle) [inline files/en-us/web/html/element/select/index.md:523]: i think MDN is using arrow functions.

**Review by estelle — COMMENTED** [review]
The edits should likely be done to the other file not to this one as the original better meets MDN coding style guidelines. Closing

**Comment by qibing147147** [comment]
> The edits should likely be done to the other file not to this one as the original better meets MDN coding style guidelines. Closing

I copy the code from this file, but It can not run correctly. So i copy the code from [https://github.com/mdn/html-examples/blob/master/custom-select/index.html](https://github.com/mdn/html-examples/blob/master/custom-select/index.html). And it can run correctly.

**Pull request closed without merging**